### PR TITLE
Avoid passing named_keys into runtime::call_host_ methods.

### PR DESCRIPTION
**Pull request by nilz3ro**
In this PR I have removed named keys from the signatures of the following functions:
* `Runtime::call_host_mint()`
* `Runtime::call_host_auction()`
* `Runtime::call_host_handle_payment()`

### CODE REVIEW

File: execution_engine/src/core/execution/executor.rs [hunk 1]
```diff
@@ -676,7 +673,7 @@ impl DirectSystemContractCall {
         &self,
         mut runtime: Runtime<R>,
         protocol_version: ProtocolVersion,
-        named_keys: &mut NamedKeys,
+        _named_keys: &mut NamedKeys,
```
Comment (darthsiroftardis) [inline execution_engine/src/core/execution/executor.rs]: Why not remove this argument if its unused?

File: execution_engine/src/core/runtime/mod.rs [hunk 2]
```diff
@@ -57,6 +57,8 @@ use crate::{
 
 use self::host_function_flag::HostFunctionFlag;
 
+use super::tracking_copy::TrackingCopyExt;
```
Comment (darthsiroftardis) [inline execution_engine/src/core/runtime/mod.rs]: Import in the wrong place, can be grouped alongside with the other core imports

**Review by darthsiroftardis — APPROVED** [review]
Looks good

**Review by mpapierski — APPROVED** [review]
LGTM but you need to fix clippy issues @nilz3ro

**Review by Fraser999 — APPROVED** [review]

**Pull request merged**